english/explore-outside/talk-child/talk-child.md

```diff
@@ -1,11 +1,13 @@
 You walk towards the green child fishing and yell out, "Hello!".
-The child looks towards you with red glaring eyes.  
+The child looks towards you with red glaring eyes.
 
 Do you:
 
 [Decide to head home because red eyes are creepy for reals.](../journey-to-flaming-tower/home/home.md)
 
-[Take your cellphone out, call Tank and ask for 
+[Take your cellphone out, call Tank and ask for
 the nearest exit out of the Matrix.](../journey-to-flaming-tower/exit/exit.md)
 
 [Try a MacGyver escape move with items in pocket.](../../macgyver/macgyver.md)
+
+[Flash the sweetest smile to the child?](../smile/smile-to-child.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.